yt/frontends/enzo_p/fields.py

```diff
@@ -40,6 +40,14 @@ class EnzoPFieldInfo(FieldInfoContainer):
     def __init__(self, ds, field_list, slice_info=None):
         super().__init__(ds, field_list, slice_info=slice_info)
 
+    def setup_fluid_fields(self):
+        super().setup_fluid_fields()
+        self.alias(
+            ("gas", "total_energy"),
+            ("gas", "specific_total_energy"),
```

Comment:
I wonder if `total_specific_energy` would be more intuitive (to me it is, but that's very low level statistics haha)

Reply:
makes senses, thank you